packages/core-bridge/Cargo.toml

```diff
@@ -19,3 +19,8 @@ tokio = "1.4.0"
 once_cell = "1.7.2"
 # TODO: revert back to version *, can't now because of this error: prerelease package needs to be specified explicitly
 temporal-sdk-core = { version = "0.1.0-alpha.1", path = "./sdk-core" }
+temporal-sdk-core-protos = { version = "0.1.0", path = "./sdk-core/sdk-core-protos" }
+
+# Local testing w/ changes to sdk-core
+#temporal-sdk-core = { version = "0.1.0-alpha.1", path = "../../../sdk-core" }
```

Comment:
There should be a mechanism for overriding this, maybe https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#paths-overrides?
Or maybe with an env var?

Reply:
The path overrides thing works, but then you've still got to toggle them in there when switching back and forth... Kinda same difference either way